### PR TITLE
Upgrade/800/logical interconnects

### PR DESCRIPTION
**Pull request by sijeesh**
### Description
Upgraded Logical Interconnect resource to support OneView API version 800

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -272,34 +272,35 @@
...
+|<sub>/rest/logical-interconnects/{id}/port-monitor</sub>                                 | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/{id}/telemetry-configurations/{tcId}</sub>              | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/{id}/telemetry-configurations/{tcId}</sub>              | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/compliance</sub>                                        | POST     | :heavy_multiplication_x:       | :heavy_multiplication_x:       | :heavy_multiplication_x:       |:heavy_multiplication_x:     |:heavy_multiplication_x:     |
+|<sub>/rest/logical-interconnects/{id}</sub>                                              | PATCH    | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |  :heavy_minus_sign:  |  :heavy_minus_sign:  |
```
Comment (AnushaY1916) [inline endpoints-support.md]: PATCH is supported in 800. Please check once?

File: examples/logical_interconnects.py [hunk 2]
```diff
@@ -80,44 +77,32 @@
...
 
+# Get scope to be added
+print("\nGet the scope that matches the specified name.")
+scope = oneview_client.scopes.get_by_name(scope_name)
+
 # Performs a patch operation
-# Endpoint supported only in api-versions 500 and below.
-if scope and (oneview_client.api_version <= 500):
+# Endpoint not supported in API version 600.
+if scope and oneview_client.api_version != 600:
```
Comment (AnushaY1916) [inline examples/logical_interconnects.py]: What about version 200

File: endpoints-support.md (hunk 1 above)
Comment (sijeesh) [reply endpoints-support.md]: Fixed it.

File: examples/logical_interconnects.py (hunk 2 above)
Comment (sijeesh) [reply examples/logical_interconnects.py]: Fixed it.

**Pull request merged**